API/chat/tests.py

```diff
@@ -670,13 +617,9 @@ def test_delete_message_that_does_not_exist(self):
             timestamp=timestamp
         )
 
-        qstring = urllib.urlencode({
-            'id': message.id + 1
-        })
-
         response = self.client.delete(
-            reverse('chat:message', args=(self.channel.name,)),
-            qstring
+            reverse('chat:message', args=(message.id + 1,)),
+            {}
```

Comment:
I disagree with the different URL naming proposal. I like the "directory-structure" URLs where URLs function as folders with files. "Files" as resources should be deleted from the same end-points they are created or listed.